src/commands/join.cpp

```diff
@@ -36,26 +37,34 @@ void Server::joinChannel(int fd, const std::string &param) {
 #ifdef DEBUG
   {
     std::ostringstream oss;
-    oss << "JOIN: client.getChannelsCount(): " << client.getChannelsCount();
+    oss << "client.getChannelsCount(): " << client.getChannelsCount();
     printLog(DEBUG_LOG, COMMAND, oss.str());
   }
   {
     std::ostringstream oss;
-    oss << "JOIN: gConfig->getLimitchanlimit: " << gConfig->getLimit(CHANLIMIT);
+    oss << "gConfig->getLimitchanlimit: " << gConfig->getLimit(CHANLIMIT);
```

Comment:
Please keep `JOIN:` keyword in log message to know what command this message comes from.